nipype/pipeline/plugins/multiproc.py

```diff
@@ -100,3 +177,126 @@ def _report_crash(self, node, result=None):
 
     def _clear_task(self, taskid):
         del self._taskresult[taskid]
+
+    def _submit_job(self, node, updatehash=False):
+        self._taskid += 1
+        try:
+            if node.inputs.terminal_output == 'stream':
+                node.inputs.terminal_output = 'allatonce'
+        except:
```

Comment:
Thanks. I don't see why this should be in a try catch. I would vote to remove it. @satra ?